CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 <!-- All new changelog items should come here -->
 
+- [#470]
+  - **Description:** Fix bug and add test guard in MediaQuery implementation
+  - **Products impact:** none
```

Comment:
Here, the product impact would be bugfix. Here's an example of release notes for `useKResponsiveWindow`. We'd ideally be able to read this from the changelog rather than opening a PR. As mentioned below, no action needed.

![Screenshot from 2023-10-23 13-34-29](https://github.com/learningequality/kolibri-design-system/assets/13509191/8487ab8b-697e-4f97-a457-cf31dfa10a8b)